docker/ceph/set-start-env.sh

```diff
@@ -17,6 +17,10 @@ if [[ "$RGW_MULTISITE" == 1 ]]; then
     export RGW=0  # Required to prevent vstart from starting any rgw daemon.
 fi
 
+if [[ "$CEPHFS_MIRROR" == 1 ]]; then
+    dnf install -y cephfs-mirror
+fi
```

Comment:
actually you can just add it to the ceph-rpm Dockerfile so that whenever a new rpm image is there it'll always have cephfs-mirror rpm installed.

https://github.com/rhcs-dashboard/ceph-dev/blob/main/docker/ceph/rpm/Dockerfile#L37

which is actually faster. we don't need a toggle around it. its fine to have it always there. we can add a toggle when we actually automate the mirroring workflow here.

Reply:
Oh okay, makes sense. I'll update it.

Reply:
Updated it @nizamial09 , directly installing the pacakge